### ISSUE TITLE
nginx: [emerg] no "events" section in configuration

### ISSUE THREAD

**Issue by fwerner13** [sherifabdlnaby/symdocker#2]
Hi, 

I encountered the following error when trying to start: 

```
web_1  | 2019/10/06 08:12:13 [emerg] 1#1: no "events" section in configuration
web_1  | nginx: [emerg] no "events" section in configuration
symdocker_web_1 exited with code 1
```

Steps to reproduce: 

- Create composer.json with a dependency so the build script does not stop when trying to copy composer.json and composer.lock
- `make build`
- `make up`

**Comment by sherifabdlnaby** [comment]
@fwerner13 I am unable to reproduce this bug. also, the "events" section is in the configuration. are you sure you did not modify it? Please reopen the issue if the problem still persists. Thanks!

**Comment by atierant** [comment]
Hi :) I reproduce the same error.
Env : Docker for Windows, docker & docker-compose installed on an Ubuntu 20.04 on WSL1.
I started with the symfony --demo app, adding the 4 files/folders, and then make up.
In docker-nginx-entrypoint, I added a `ls -la /etc/nginx/nginx.conf` before starting nginx.
`-rwxrwxrwx 1 root root 0 Sep 28 16:53 /etc/nginx/nginx.conf`
In the Dockerfile, if I place a `RUN ls -la /etc/nginx/nginx.conf` just after the copy, the file appears to be correctly copied. But then, it is not.  
So if the problem is later, it seems to be a [DockerForWindows issue](https://github.com/docker/for-win/issues/4746).... The file remains empty so no "events" section.
